### PR TITLE
Image changes - Red text for answers

### PR DESCRIPTION
**Pull request by warrenbailey**
Porting the changes from https://github.com/ONSdigital/sdx-transform-cora/pull/22

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ### Unreleased
+
```
Comment (JamesGardiner) [inline CHANGELOG.md]: No space

File: transform/transformers/PDFTransformer.py [hunk 2]
```diff
@@ -28,22 +30,32 @@
...
+style_ssh = styles["Heading3"]
+style_ssh.alignment = TA_LEFT
 
 # Main heading style
 styleH = styles['Heading1']
 styleH.alignment = TA_CENTER
 
+
+# Answer Style
+style_answer = copy(styles["BodyText"])
```
Comment (JamesGardiner) [inline transform/transformers/PDFTransformer.py]: Do you definitely want a shallow copy here? What's the underlying data structure of `styles["BodyText"]`?

File: CHANGELOG.md (hunk 1 above)
Comment (warrenbailey) [inline CHANGELOG.md]: I'll remove

File: transform/transformers/PDFTransformer.py (hunk 2 above)
Comment (warrenbailey) [inline transform/transformers/PDFTransformer.py]: To be honest, I just copied it from the other review. I'll look into it.

Comment (warrenbailey) [inline transform/transformers/PDFTransformer.py]: The underlying structure is s a class containing 2 dicts

**Comment by warrenbailey** [comment]
Made a similar change to sdx-transform-cora https://github.com/ONSdigital/sdx-transform-cora/pull/28

**Pull request merged**